### PR TITLE
[TASK] Add some `todo` markers and comments

### PR DESCRIPTION
**Pull request by sbuerk**
This change adds a couple of `@todo` and
comments to several places to emphasises
where further thinking needs to be done.

Releases: main

### CODE REVIEW

File: Classes/Domain/Repository/GlossaryRepository.php [hunk 1]
```diff
@@ -408,7 +410,7 @@ private function getAvailableLocalizations(int $pageId): array
 
     protected function getTargetLanguageIsoCode(Site $site, int $languageId): string
     {
-        // TODO add support for deprecated sys_language table
+        // @todo Add support for deprecated sys_language table.
```
Comment (calien666) [inline Classes/Domain/Repository/GlossaryRepository.php:413]: TODO can be removed, as v4 won't support sys_language anymore

Comment (sbuerk) [reply Classes/Domain/Repository/GlossaryRepository.php:413]: That was a sidechange/adjustment. Can we let it be as is ? We need to go through all classes anyway and I guess more cleanups will follow - or am I wrong ?

Comment (sbuerk) [reply Classes/Domain/Repository/GlossaryRepository.php:413]: Forget, will remove it.

Comment (sbuerk) [reply Classes/Domain/Repository/GlossaryRepository.php]: done.

**Review by calien666 — APPROVED** [review]

**Pull request merged**